README.md

```diff
@@ -39,19 +39,14 @@ With the ESnext field declarations proposal, the above example can be written as
 class Counter extends HTMLElement {
```

Comment:
Oh, my understanding was that they would be functionally equivalent in React code. The docs mention being able to use the "experimental property initializer syntax" which I thought referred to an earlier version of this proposal.

https://facebook.github.io/react/docs/handling-events.html